requirements.txt

```
@@ -1 +1 @@
-orionsdk>=0.0.3,<0.1
+orionsdk
```

Instead of removing the pin, it would be better to change it so that it'll auto update 0.x.y and not pull in 1.0.0.

```suggestion
orionsdk~=0.3
```

---

Makes sense.  Suggested limit added to the requirement.txt file